### PR TITLE
Spanish Translation

### PR DESCRIPTION
**Pull request by Nemesti**

### CODE REVIEW

**Comment by Nemesti** [comment]
It's the first time I use GITHUB, I hope everything is fine.

**Comment by StefArma** [comment]
Oh wow! i'm amazed!

File: AntistasiOfficial.Altis/Stringtable.xml [hunk 1]
```diff
@@ -1,4510 +1,5005 @@
...
-				<Original>We know an %4 is stationed in %1. This is a good chance to steal or destroy it before it causes more damage. Do it before %2:%3.</Original>
-				<Russian>Мы узнали, что %4 дислоцируется в %1. Сейчас самое время, чтобы захватить или уничтожить его, прежде чем он сможет наделать бед. Задание должно быть выполнено до %2:%3.</Russian>
-			</Key>
-			<Key ID="STR_TSK_TD_FNDCIV">
-				<Original>Contact the Corrupt Politician</Original>
-				<Russian>Контакт с политиком</Russian>
-			</Key>
-			<Key ID="STR_TSK_TD_DESC_FNDCIV">
-				<Original>A local politician who is sympathetic to our cause is willing to meet us near %1 until %2:%3. As it happens, he has a couple of jobs he's offering to undermine %4 control in this AO. He has stated that he is willing to speak to us only on the condition that none of our agents are overt within that town. In other words, put on your sunday-best and meet him disguised. </Original>
-				<Russian>Местный политик, симпатизирующий нам, готов встретиться с нами рядом с %1 до %2:%3. Как это бывает, у него есть несколько заданий, которые, как он предполагает, смогут лишить %4 контроля в этом районе. Он заявил, что готов говорить с на
```
Comment (StefArma) [inline AntistasiOfficial.Altis/Stringtable.xml]: there are theese + that has to be deleted

Comment (StefArma) [inline AntistasiOfficial.Altis/Stringtable.xml]: thhere are theese lines which were added wrongly, need to be removed

**Comment by StefArma** [comment]
uhmm for some reasons i quoted the wrong part of code... anyway check for extra
+
@@  @@

**Comment by Nemesti** [comment]
I should have copied something wrong when I added the code. I check and try to fix it.

**Comment by Nemesti** [comment]
I think that is correct, but don't know if u can see the changes.

File: AntistasiOfficial.Altis/Stringtable.xml [hunk 2]
```diff
@@ -1581,724 +1957,894 @@
...
 		     <Key ID="STR_HINTS_TELEPORT">
 			    <Original>Click on the position you wish to teleport to.</Original>
 				<Russian>Кликните по зоне, куда Вы хотите телепортироваться.</Russian>
+			     	<Spanish>Click en la posición a la que deseas ser teletransportado.</Spanish>
 			</Key>
 		</Container>
 		<Container name="GARBAGE_CLEAR">
-			<Key ID="STR_HINTS_GARB_CLR">
-				<Original>Deleting Garbage. Please wait</Original>
+		     <Key ID="STR_HINTS_GARBAGE_CLR">
```
Comment (AlexTriada) [inline AntistasiOfficial.Altis/Stringtable.xml]: you can not do this

File: AntistasiOfficial.Altis/Stringtable.xml [hunk 3]
```diff
@@ -2314,1032 +2860,1294 @@
...
 				<Original>Manage Garrisons</Original>
 				<Russian>Гарнизоны</Russian>
+				<Spanish>Admin. Guarnición</Spanish>
 			</Key>
 			<Key ID="STR_D_MOVE_HQ_TAZ">
 				<Original>Move HQ to another Zone</Original>
 				<Russian>Переезд Штаба</Russian>
+				<Spanish>Mover HQ a otra zona</Spanish>
 			</Key>
 			<Key ID="STR_D_INGAME_ML">
```
Comment (AlexTriada) [inline AntistasiOfficial.Altis/Stringtable.xml]: dubbing ID="STR_D_INGAME_ML" in 2880 and 3882

File: AntistasiOfficial.Altis/Stringtable.xml (hunk 2 above)
Comment (Nemesti) [reply AntistasiOfficial.Altis/Stringtable.xml]: Ok, I see what u say. Thx, change this part

File: AntistasiOfficial.Altis/Stringtable.xml [hunk 4]
```diff
@@ -2314,1032 +2860,1294 @@
...
 				<Russian>Музыка Вкл/Выкл</Russian>
+				<Spanish>Música SI/NO</Spanish>
 			</Key>
 			<Key ID="STR_D_GAME_OP">
 				<Original>Ingame Member's List</Original>
 				<Russian>Список игроков</Russian>
+				<Spanish>Lista Miembros Jugando</Spanish>
+			</Key>
+			<Key ID="STR_D_INGAME_ML">
+				<Original>Maintenance Menu</Original>
```
Comment (AlexTriada) [inline AntistasiOfficial.Altis/Stringtable.xml]: where did you get this from?

File: AntistasiOfficial.Altis/Stringtable.xml (hunk 3 above)
Comment (AlexTriada) [reply AntistasiOfficial.Altis/Stringtable.xml]: all is OK

**Comment by AlexTriada** [comment]
Looks like everything is OK

**Comment by StefArma** [comment]
Wow awsome!!!

**Pull request merged**